.jscsrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Can't you use one of the existing presets as a starting point? That should massively reduce the amount of config here.